src/components/timecard/shift/Shift.jsx

```diff
@@ -15,14 +14,19 @@ import { validateServiceErrors } from '../../../utils/api-utils/ApiUtils';
 import { useApplicationContext } from '../../../context/ApplicationContext';
 import dayjs from 'dayjs';
 
-const EditShiftTimecard = ({
+const Shift = ({
```

Comment:
The props seem to be highlighting an issue for me on dependencies in the way the components have been created

Reply:
To be resolved in future work